Gemfile

```diff
@@ -51,11 +51,11 @@ gem '3scale_time_range', '0.0.6'
 gem 'statsd-ruby', require: false
 
 # Sidekiq
-gem 'sidekiq', '~> 6.4.0', require: %w[sidekiq sidekiq/web]
-gem 'sidekiq-batch'
+gem 'sidekiq', '~> 7', require: %w[sidekiq sidekiq/web]
+gem 'sidekiq-batch', github: '3scale/sidekiq-batch', branch: 'redis-client'
 gem 'sidekiq-cron', require: %w[sidekiq/cron sidekiq/cron/web]
 gem 'sidekiq-lock'
-gem 'sidekiq-throttled'
+gem 'sidekiq-throttled', '~> 1.0.0.alpha.1'
```

Comment:
Not especially happy using an alpha but it's the only available version supporting Sidekiq 7

Reply:
In rubygems most releases are alpha anyway...